nameresolution/mdns/mdns.go

```diff
@@ -373,7 +659,10 @@ func (m *resolver) refreshAllApps(ctx context.Context) error {
 		go func(a string) {
 			defer wg.Done()
 
-			m.refreshApp(ctx, a)
+			err := m.refreshApp(ctx, a)
+			if err != nil {
+				m.logger.Warnf("error refreshing mDNS addresses for app id %s: %v", a, err)
```

Comment:
Good question, so assuming the reason we can't refresh the app causes every app ID refresh to fail. You'd log once per app ID every 30 seconds. Depending how you monitor this might be annoying behaviour but I don't think it'd flood the log. Maybe it should be an aggregated error using `uber.go/multierr` to avoid multiple log lines.